examples/dmd/wave_equation.cpp

```diff
@@ -0,0 +1,621 @@
+ /*
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+//
+//                       libROM MFEM Example: Wave_Equation (adapted from ex23.cpp)
+//
+// Compile with: make wave_equation 
+//
+// =================================================================================
+//
+// Sample runs and results for Time-Windowing DMD:
+//
+// Command 1:
+//  wave_equation -o 4 -tf 5 -nwinsamp 25 
+//
+// Output 1:
+// Relative error of DMD solution (u) at t_final: 5 is 3.0483662e-05
+// Elapsed time for solving FOM: 3.122185e+00 second
+// Elapsed time for training DMD: 6.904051e-01 second
+// Elapsed time for predicting DMD: 2.496171e-03 second
+//
+// Command 2:
+// wave_equation -o 4 -tf 5 -nwinsamp 25 -visit
+// 
+// Output 2: 
+// Relative error of DMD solution (u) at t_final: 5 is 3.0483662e-05
+// Elapsed time for solving FOM: 3.239955e+00 second
+// Elapsed time for training DMD: 7.438771e-01 second
+// Elapsed time for predicting DMD: 1.225844e+00 second
```

Comment:
DMD prediction is much faster than this without `-visit`. This raises the question of how to compare run times fairly. VisIt output for such a small test can severely affect run time, so I think performance should be compared without any file output. Of course, there is logic to do the DMD predictions at all steps only if there is VisIt output, which unfairly compares all the FOM timesteps against DMD prediction at only 1 point in time (the final time). I'm not sure what is best to do here, but at least I think `-visit` should not be included when timing performance.

Reply:
@chldkdtn Do you have an opinion on timing values for this example accounting for the `-visit` option?

Reply:
Not using -visit option when timing performance sounds good to me.

Reply:
Agreed, DMD performance results with the `-visit` flag are now removed.